README.md

```diff
@@ -19,41 +19,33 @@
 
 - [Project presentation](#project-presentation)
 - [Getting Started](#getting-started)
-	- [Contents overview](#contents-overview)
 	- [Installation](#installation)
 	- [Contributing](#contributing)
 - [References](#references)
 - [Funding](#funding)
 
 ## Project presentation
 
-Neuroimaging workflows are highly flexible, leaving researchers with multiple possible options to analyze a dataset [(Carp, 2012)](https://www.frontiersin.org/articles/10.3389/fnins.2012.00149/full).
-However, different analytical choices can cause variation in the results [(Botvinik-Nezer et al., 2020)](https://www.nature.com/articles/s41586-020-2314-9), leading to what was called a "vibration of effects" [(Ioannidis, 2008)](https://pubmed.ncbi.nlm.nih.gov/18633328/) also known as analytical variability. 
-
 **The goal of the NARPS Open Pipelines project is to create a codebase reproducing the 70 pipelines of the NARPS project (Botvinik-Nezer et al., 2020) and share this as an open resource for the community**. 
 
-To perform the reproduction, we are lucky to be able to use the [descriptions provided by the teams](https://github.com/poldrack/narps/blob/1.0.1/ImageAnalyses/metadata_files/analysis_pipelines_for_analysis.xlsx).
-We also created a [shared spreadsheet](https://docs.google.com/spreadsheets/d/1FU_F6kdxOD4PRQDIHXGHS4zTi_jEVaUqY_Zwg0z6S64/edit?usp=sharing) that can be used to add comments on pipelines (e.g.: identify the ones that are not reproducible with NiPype).
+We base our reproductions on the [original descriptions provided by the teams](https://github.com/poldrack/narps/blob/1.0.1/ImageAnalyses/metadata_files/analysis_pipelines_for_analysis.xlsx) and test the quality of the reproductions by comparing our results with the original results published on NeuroVault.
+
+:vertical_traffic_light: See [the pipeline dashboard](https://github.com/Inria-Empenn/narps_open_pipelines/wiki/pipeline_status) to view our current progress at a glance.
 
-:vertical_traffic_light: Lastly, please find [here in the project's wiki](https://github.com/Inria-Empenn/narps_open_pipelines/wiki/pipeline_status) a dashboard to see pipelines work progresses at first glance.
+*Botvinik-Nezer, Rotem, et al. "Variability in the analysis of a single neuroimaging dataset by many teams." Nature 582.7810 (2020): 84-88.*
 
-## Getting Started
+## Contributing
 
-### Contents overview
+NARPS open pipelines uses nipype as a workflow manager and provides a series of templates and examples to help reproduce the different teams’ analysis. 
```

Comment:
I was trying to keep the project section very short and I think it might make sense to keep this in Contributing (at least for now?). Added the link to nypipe website ➡️ https://github.com/cmaumet/narps_open_pipelines/commit/24a6dbe303571eabd23774ca834b1538dc7ee56e